code/datums/uplink_item.dm

```diff
@@ -1440,8 +1442,11 @@ var/list/uplink_items = list()
 			continue
 		bought_items += I.item
 		remaining_TC -= I.cost
+		itemlog += I.name // To make the name more readable for the log compared to just i.item
 
 	U.purchase_log += "<BIG>[bicon(C)]</BIG>"
 	for(var/item in bought_items)
 		new item(C)
 		U.purchase_log += "<BIG>[bicon(item)]</BIG>"
+	var/mob/user = usr
```

Comment:
Don't do this. If the code uses `usr` anyways, just put `usr` in the key_name argument.